src/ui/common/src/components/pages/integrations/index.tsx

```diff
@@ -98,108 +98,104 @@ const IntegrationsPage: React.FC<Props> = ({
-                  </pre>
-                </Box>
-              </Box>
-            )}
-          </Box>
+      <Box paddingBottom={4}>
+        <Typography variant="h5" marginBottom={2}>
+          Available Resources
+        </Typography>
+        <ConnectedIntegrations
+          user={user}
+          forceLoad={forceLoad}
+          connectedIntegrationType={ConnectedIntegrationType.Compute}
+        />
+        <ConnectedIntegrations
+          user={user}
+          forceLoad={forceLoad}
+          connectedIntegrationType={ConnectedIntegrationType.Data}
+        />
+        <ConnectedIntegrations
+          user={user}
+          forceLoad={forceLoad}
+          connectedIntegrationType={ConnectedIntegrationType.Other}
+        />
 
-          <Box marginY={2}>
-            <Divider />
-          </Box>
-
-          <Typography variant="h5" marginY={2}>
-            Add New Resources
-          </Typography>
-
-          <Typography variant="h6" marginY={2}>
-            Compute
-          </Typography>
-          <AddIntegrations
-            user={user}
-            category={IntegrationCategories.COMPUTE}
-            supportedIntegrations={SupportedIntegrations}
-          />
-          <Typography variant="h6" marginY={2}>
-            Data
-          </Typography>
-          <AddIntegrations
-            user={user}
-            category={IntegrationCategories.DATA}
-            supportedIntegrations={SupportedIntegrations}
-          />
+        <Box>
           <Typography variant="h6" marginY={2}>
-            Container Registry
+            Artifact Storage
           </Typography>
-          <AddIntegrations
-            user={user}
-            category={IntegrationCategories.CONTAINER_REGISTRY}
-            supportedIntegrations={SupportedIntegrations}
-          />
+          <MetadataStorageInfo serverConfig={serverConfig.config} />
+          {!isLoading && lastFailedFormattedTimestamp && (
+            <Box>
+              <Typography
+                variant="body2"
+                fontWeight="fontWeightRegular"
+                marginTop={2}
+                marginBottom={1}
+              >
+                The last artifact storage migration, which started at{' '}
+                {lastFailedFormattedTimestamp}, has failed! As a result, the
+                artifact storage has not changed from `
+                {serverConfig.config?.storageConfig.integration_name}`.
+              </Typography>
+              <Box
+                sx={{
+                  borderRadius: 2,
+                  backgroundColor: theme.palette.red[100],
+                  color: theme.palette.red[600],
+                  p: 2,
+                  paddingBottom: '16px',
+                  paddingTop: '16px',
+                  height: 'fit-content',
+                }}
+              >
+                <pre style={{ margin: '0px' }}>
```

Comment:
You might be able to move this to the sx props above instead.
https://www.w3schools.com/tags/tag_pre.asp